engine/src/test/groovy/graphql/nadel/engine/ServiceExecutionParametersTest.groovy

```diff
@@ -41,5 +45,7 @@ class ServiceExecutionParametersTest extends Specification {
         parameters.hydrationCall
         parameters.hydrationDetails.getTimeout() == 10
         parameters.hydrationDetails.getBatchSize() == 50
+        parameters.hydrationDetails.getHydrationSourceService() == sourceService
+        parameters.hydrationDetails.getHydrationSourceField() == coords
     }
 }
```

Comment:
Question - in the new KT tests - how do I test this is passed in in an execution sense